packages/kbn-search-connectors/types/native_connectors.ts

```diff
@@ -157,13 +157,33 @@ export const NATIVE_CONNECTOR_DEFINITIONS: Record<string, NativeConnector | unde
         validations: [],
         value: '',
       },
+      containers: {
+        default_value: null,
+        depends_on: [],
+        display: DisplayType.TEXTAREA,
+        label: i18n.translate(
+          'searchConnectors.nativeConnectors.azureBlobStorage.containerNameLabel',
+          {
+            defaultMessage: 'List of Containers',
```

Comment:
Why the label is different from [here](https://github.com/elastic/connectors/blob/f70ebb9d8718a57cbfc7dd1a0e4be22667d0aefc/connectors/sources/azure_blob_storage.py#L79).

And you should also consider update labels for `account_name`, `account_key` and `blob_endpoint`